### PR TITLE
Add Circle CI config for testing

### PR DESCRIPTION
**Pull request by cpcloud**

### CODE REVIEW

**Comment by wesm** [comment]
Hey, I just turned on Circle CI for this repo (Circle CI requires extremely invasive permissions, so I created a special https://github.com/ibis-bot user which only has admin / push/pull on the ibis repo), so in theory when the circle.yml hits master builds should be triggered

https://circleci.com/gh/cloudera/ibis

**Comment by cpcloud** [comment]
Cool. Thanks for doing that. Do you think it makes sense to land a version that has tests for sqlite and postgres and follow up with testing for impala? There's a bit of additional work to get the docker image for impala working that I don't think should slow down having at least some testing.

**Comment by wesm** [comment]
That's fine by me.

**Comment by wesm** [comment]
You might try touching this commit to see if it triggers a Circle CI build; if not I can try twiddling the settings to get it to work

**Comment by wesm** [comment]
I turned on permissive PR builds. Note the security implications of this detailed here: https://circleci.com/docs/fork-pr-builds/. Because of this, it's not safe to put any secrets (keys, sensitive env variables, etc.) in the Circle CI config.

**Comment by cpcloud** [comment]
Looks like it's triggering. I'll make a TODO issue for things that we'll want to eventually add to the CI. Top two are Impala testing and multiple Python versions. The latter of which I just haven't tinkered with enough on CircleCI.

File: ibis/impala/metadata.py [hunk 1]
```diff
@@ -43,15 +43,15 @@ def _converter(tup):
...
+        ts = pd.Timestamp(x, tz='UTC')
+        return ts.to_pydatetime().replace(tzinfo=None)
     except (ValueError, TypeError):
         return x
 
 
 def _try_unix_timestamp(x):
     try:
-        ts = pd.Timestamp.fromtimestamp(int(x))
+        ts = pd.Timestamp.utcfromtimestamp(int(x))
```
Comment (cpcloud) [inline ibis/impala/metadata.py]: @wesm Just want to call your attention to these changes. This ensures that timestamps look the same no matter what timezone Python is run in. I actually only discovered this when I was in NY a while ago.

Comment (wesm) [inline ibis/impala/metadata.py]: Good catch. No problem

**Comment by wesm** [comment]
Sweet, we should add a badge to the README

**Comment by cpcloud** [comment]
Yep. Will do tonight.

**Pull request closed without merging**